test/tools/CreateChildProcess/CreateChildProcess.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading;
+
+namespace CreateChildProcess
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            if (args.Length > 0)
+            {
+                uint num = UInt32.Parse(args[0]);
+                for (uint i = 0; i < num; i++)
+                {
+                    Process child = new Process();
+                    child.StartInfo.FileName = Process.GetCurrentProcess().MainModule.FileName;
```

Comment:
Need this as a native executable to execute a particular code path in powershell for handing native commands